### ISSUE TITLE
Pivot: Narrator focus changes in a strange way on Pivot selection

### ISSUE THREAD

**Issue by wygoodin** [OfficeDev/office-ui-fabric-react#3987]
### Bug Report

- __Package version(s)__: 5.47.0
- __Browser and OS versions__: Edge with Narrator, Windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) 
No (I have made several attempts to fix, and can't figure it out)

Requested priority: (Blocking, High, Normal, Low)
High (eventually blocking)

#### Describe the issue:

When you select a pivot using Narrator scan mode, the focus of narrator seems to be on some element related to that pivot link, but not the regular button, as if you hit up it takes you two pivot links backwards, and if you hit down, it reselects the link you're on rather than moving forwards.

#### Actual behavior:

Movement operations are off by one after selection

#### Expected behavior:

Movements operations are consistent regardless of selection actions.

### If applicable, please provide a codepen repro:

This can be reproed on the regular office fabric examples page for Pivots

**Comment by Jahnp** [comment]
@jspurlin @cschlechty -- thoughts on this? Is there something we're doing wrong here?

**Comment by jspurlin** [comment]
I just played with the pivot component on the external example pivot page and navigating with scan mode on or off seemed to work as expected.

@wygoodin do you have more details (repro steps) for the issue you are describing?

**Comment by jspurlin** [comment]
@wygoodin this seems like a Narrator bug, if after hitting ENTER with "share with me" selected you then press DownArrow before hitting UpArrow focus goes to the expected location. Closing this issue and there should be a follow up with the Narrator team to track this issue on their end
